Update README.md

DardanBala opened the pull request:
Added simple example how to implement the ListNotifier.

DardanBala committed:
Update README.md

Added simple example how to implement the ListNotifier.

escamoteur commented on README.md:
```diff
@@ -11,4 +11,107 @@ planed are:
+                builder: (context, value, child) => Column(
+                  mainAxisSize: MainAxisSize.max,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: List.from(_list.map((value) => Text('List Item: $value'))),
```
Why did you use List.from and not just add a ´toList()` at the end?

DardanBala replied:
I used to List.from to generate list of widgets (in this case list of Text widget). Couldn't use `_list.toList()` because `_list` its type of `<String>`

escamoteur replied:
`_list.map<Widget>((value) => Text('List Item: $value').toList()`
should work because it's after the Map

DardanBala replied:
Yep its working, check my latest commit.

DardanBala committed:
Update README.md

Changed the way how to display the children of the column, from List.from to toList()
